finagle-core/src/main/scala/com/twitter/finagle/socks/flags.scala

```diff
@@ -8,6 +8,9 @@ private[finagle] object socksProxyHost extends GlobalFlag("", "SOCKS proxy host"
 private[finagle] object socksProxyPort extends GlobalFlag(0, "SOCKS proxy port") {
   override val name = "socksProxyPort"
 }
+private[finagle] object socksProxyForLocalhost extends GlobalFlag(false, "If true SOCKS proxy is also use for localhost") {
```

Comment:
its nice when we state the default. how about something like,
"Bypass the SOCKS proxy for localhost connections, defaults to true."?